generated/nidaqmx/_task_modules/triggering/start_trigger.py

```diff
@@ -1037,6 +1037,20 @@ def cfg_dig_pattern_start_trig(
         self._interpreter.cfg_dig_pattern_start_trig(
             self._handle, trigger_source, trigger_pattern, trigger_when.value)
 
+    def cfg_time_start_trig(self, when, timescale=Timescale.USE_HOST):
+        """
+        New Start Trigger
+
+        Args:
+            when (nidaqmx.constants.DateTime): Specifies when to
```

Comment:
I don't think this is the right type. I added [AbsoluteTime in _lib_time.py](https://github.com/ni/nidaqmx-python/blob/c5fddf5c9dd6be9b0d02bcd96338963176515cf3/src/handwritten/_lib_time.py) that we should accept. We should also accept a `datetime` and a `hightime` from the std library.

Reply:
The interpreter interface should only use datetime/hightime, not the ctypes and gRPC specific types.

The interpreter implementations should be responsible for converting datetime/hightime to/from AbsoluteTime/google.protobuf.Timestamp.

Reply:
Yup, you're right. I'm just remembering everything we talked about when I added `AbsoluteTime`. `AbsoluteTime` is the type that the lib_interpreter uses to interface with the CAPI. `google.protobuf.Timestamp` for the grpc_interpreter. In addition, the top-level API should accept `datetime` or `hightime` as well. So just pass them down to the interpreter and each will convert to their respective types.

Reply:
Some starting points:
- Library
  - https://github.com/ni/nidaqmx-python/blob/master/src/codegen/templates/library_interpreter/default_c_function_call.py.mako
  - https://github.com/ni/nidaqmx-python/blob/master/src/codegen/utilities/interpreter_helpers.py#L135
- gRPC
  - https://github.com/ni/nidaqmx-python/blob/master/src/codegen/templates/grpc_interpreter/default_grpc_function_call.py.mako
  - https://github.com/ni/nidaqmx-python/blob/master/src/codegen/utilities/interpreter_helpers.py#L329